COPR builds are failing due to missing cap-ng.h

The COPR builds have been failing since February 1:
https://copr.fedorainfracloud.org/coprs/dcantrel/rpminspect/builds/

They're failing in compilation due to a missing header file:
```
../lib/files.c:30:10: fatal error: cap-ng.h: No such file or directory
   30 | #include <cap-ng.h>
      |          ^~~~~~~~~~
```
I assume this is due to the intended change from cap-ng to libcap that changed in the specfile but may not have changed everywhere in the code.

--- a/lib/files.c
+++ b/lib/files.c
@@ -27,7 +27,7 @@
 #include <unistd.h>
 #include <libgen.h>
 
-#include <cap-ng.h>
+#include <sys/capability.h>
 
 #include <rpm/header.h>
 #include <rpm/rpmtd.h>

--- a/lib/inspect_ownership.c
+++ b/lib/inspect_ownership.c
@@ -20,7 +20,7 @@
 #include <string.h>
 #include <errno.h>
 #include <assert.h>
-#include <cap-ng.h>
+#include <sys/capability.h>
 
 #include "rpminspect.h"
 

